secondary_bulk_insert_script.sql

```diff
@@ -70,3 +70,50 @@ INSERT INTO [dbo].[User]
 		   (2, 5, '110 527 5962', 20),
 		   (2, 1, '930 214 1784', 37);
 GO
+
+INSERT INTO dbo.[Area]
+           ([Postal_Code]
+		   ,[Suburb]
+		   ,[City]
+		   ,[Province]
+		   )
+
```

Comment:
Sorry Andrew, before you merge I see that there is some differences between your code style and I think it was Amy's. The previous code wrapped "dbo" in square brackets like, "[dbo]". And the spacing between the "VALUES" statements are different.

Reply:
Cool cool, I sorted those out

Reply:
Awesome :), thank you!